bedrock/products/templates/products/relay/includes/matrix.html

```diff
@@ -79,26 +83,55 @@
           <div class="u-hidden">{{ ftl('plan-matrix-heading-price') }}</div>
         </th>
         <td>
-            {{ ftl('plan-matrix-price-free') }}
-            <button>{{ ftl('plan-matrix-get-relay-cta') }}</button>
+          <div class="c-matrix-footer">
+            <em class="c-matrix-footer-free">{{ ftl('plan-matrix-price-free') }}</em>
+            <a href="https://relay.firefox.com/accounts/fxa/login/?process=login" rel="external noopener" class="mzp-c-button mzp-t-product mzp-t-lg">{{ ftl('plan-matrix-get-relay-cta') }}</a>
```

Comment:
Turns out we have a `relay_fxa_button` helper I should probably be using for this. I'll submit that change as a follow up PR because changing the helper requires updating tests and stuff.